### PR TITLE
Note detail page

### PR DESCRIPTION
**Pull request by hasinichathu**

### CODE REVIEW

File: lib/screens/notes_detail.dart [hunk 1]
```diff
@@ -11,9 +11,96 @@ class _NoteDatailsState extends State<NoteDatails> {
     return Scaffold(
       appBar: AppBar(
         title: Text("Note title"),
-
       ),
-      body: Text("Hi"),
+      body: Stack(
```
Comment (yashodgayashan) [inline lib/screens/notes_detail.dart:15]: Why did you used stack?

File: lib/screens/notes_detail.dart [hunk 2]
```diff
@@ -11,9 +11,96 @@ class _NoteDatailsState extends State<NoteDatails> {
...
+        children: <Widget>[
+          ListView(
```
Comment (yashodgayashan) [inline lib/screens/notes_detail.dart:17]: Why did you used two listviews instead of one?

**Review by yashodgayashan — CHANGES_REQUESTED** [review]

File: lib/screens/notes_detail.dart (hunk 1 above)
Comment (hasinichathu) [reply lib/screens/notes_detail.dart:15]: without that title and description get overlapped.

File: lib/screens/notes_detail.dart (hunk 2 above)
Comment (hasinichathu) [reply lib/screens/notes_detail.dart:17]: I'll change it

Comment (hasinichathu) [reply lib/screens/notes_detail.dart:17]: By the way one list view is for vertical scrolling and another list view is for horizontal. Can you comment on that?

Comment (yashodgayashan) [reply lib/screens/notes_detail.dart:17]: Please push after doing changes.

**Pull request merged**